src/utils/networked-pdf-schema.ts

```diff
@@ -0,0 +1,31 @@
+import { NetworkedPDF } from "../bit-components";
+import { defineNetworkSchema } from "./define-network-schema";
+import { NetworkSchema, read, StoredComponent, write } from "./network-schemas";
+import type { EntityID } from "./networking-types";
+
+const runtimeSerde = defineNetworkSchema(NetworkedPDF);
+export const NetworkedPDFSchema: NetworkSchema = {
+  componentName: "networked-pdf",
+  serialize: runtimeSerde.serialize,
+  deserialize: runtimeSerde.deserialize,
+  serializeForStorage: function serializeForStorage(eid: EntityID) {
+    return {
+      version: 1,
+      data: {
+        page: read(NetworkedPDF.page, eid)
+      }
+    };
+  },
+  deserializeFromStorage: function (eid: EntityID, component: StoredComponent) {
+    if (component.version !== 1) {
+      // Don't throw, even though it's an error. It's probably not game breaking.
+      // TODO It would be nice to have a "strict mode" for development / testing
+      // that would elevate the severity of this to throw an error.
+      console.error("Failed to deserialize stored networked-pdf component data.", component);
+      return;
+    }
+
+    const { page }: { page: number } = component.data;
+    write(NetworkedPDF.page, eid, page);
+  }
+};
```

Comment:
In https://github.com/mozilla/hubs/pull/5896/commits/07e63899bffdf0acc10a1700027702a7e0effae7 I simplified `deserializeFromStorage` so that it didn't handle migrations from one `version` to another. This is the "simpler" version, but I don't like it better. 

My argument for writing all (storable) component schemas with "builtin" support/awareness of migrations/versions is:
- If all components that will be stored in the db have version numbers and support migrations the same way, then there's one way these storable components are ever written. Anyone who wants to store data in the database needs to learn this concept and conform to the spec, so we write a doc with an example to explain it. It's a little cumbersome everytime we want to store new component data, but at least we can copy/paste from any of the other component schemas that have only 1 version supported, so the marginal inconvenience is really only an extra 5 minutes converting old property names to new property names, fixing typos etc.
- If components only _sometimes_ have version numbers and handle migrations, and most components _don't need them_, then what I expect to happen is that the Foo component has a version _string_ and the Bar component handles versions in some bespoke way and the Baz component was written by a developer who hadn't encountered the idea at all and they are writing a post asking why their scenes with old Baz components is broken. In other words, it won't be obvious that we offered a solution to the problem or wanting to change your component schema (for a stored component), so everyone invented their own way to do it.

I would like to revert that commit to restore the "complicated" implementation.